pkg/synopsysctl/cmd_migrate.go

```diff
@@ -585,6 +590,29 @@ func migrateOpsSight(namespace string) error {
 	return nil
 }
 
+// migrateSize will ensure that the Size CRD exists as well as the default sizes
+func migraSize(namespace string) error {
+	isClusterScoped := util.GetClusterScope(apiExtensionClient)
+	if _, err := util.GetCustomResourceDefinition(apiExtensionClient, util.SizeCRDName); err != nil {
+		crd, err := getCrdConfigs(namespace, isClusterScoped, []string{util.SizeCRDName})
+		if err != nil {
+			return err
+		}
+		if err := deployCrds(namespace, isClusterScoped, crd); err != nil {
+			return err
+		}
+	}
+	sizes := []string{"small", "medium", "large"}
```

Comment:
Don't we need x-large size here as well?

Also, rather than listing them here and having them defined in the size package, provide a function in the size package to get all the sizes.  Keep all the definitions and whatnot in 1 place.

Reply:
done